app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -81,13 +81,17 @@ class MultipleDisposalsTriageController @Inject() (
     with SessionUpdates {
 
   def guidance(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    println(request.userType)
```

Comment:
this looks like it was left in by mistake

Reply:
OMG. yes. Done.